src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -305,16 +305,16 @@ const FacilitySearch = props => {
               searchInputError ? 'caregiver-facilities-search-input-error' : ''
             }`}
           >
-            <label
-              htmlFor="facility-search"
+            <p
               className="vads-u-margin-top--0 vads-u-margin-bottom--1"
+              aria-hidden="true"
             >
               {content['form-facilities-search-label']}
               <span className="vads-u-color--secondary-dark"> (*Required)</span>
-            </label>
+            </p>
             {searchInputError && searchError()}
             <VaSearchInput
-              label={content['form-facilities-search-label']}
+              label={`${content['form-facilities-search-label']} (*Required)`}
```

Comment:
Let's add the `(*Required)` to the content.json. I see it used above as well. These all need to live in the content file.

Reply:
Caught me in the middle of updating it 🙂 . Good call!